lib/jekyll/commands/serve.rb

```diff
@@ -101,7 +101,11 @@ def process(opts)
           register_reload_hooks(opts) if opts["livereload"]
           setup(destination)
 
+          start_up_livereload(opts)
           start_up_webrick(opts, destination)
+
+          launch_browser @server, opts if opts["open_url"]
+          boot_or_detach @server, opts
```

Comment:
Since you're refactoring these lines, might as well clean them up..
Its a common style in Jekyll to improve readability by using parantheses to scope parameters.

```ruby
launch_browser(@server, opts) if opts["open_url"]
boot_or_detach(@server, opts)
```

Reply:
OK, I see.

I'll try it.

Reply:
Fixed by https://github.com/jekyll/jekyll/pull/6889/commits/b5b48c15efa895353e3798a4a664b5c4e4735500